content/docs/concepts/resiliency/_index.md

```diff
@@ -95,19 +95,17 @@ The following provisioning types are supported and have been tested:
 * Failing up to 3 nodes at a time in 9 worker node clusters, or failing 1 node at a time in smaller clusters. Application recovery times are dependent on the number of pods that need to be moved as a result of the failure. See the section on "Testing and Performance" for some of the details.
 * Multi-array are supported. In case of CSI Driver for PowerScale and CSI Driver for Unity, if any one of the array is not connected, the array connectivity will be false. CSI Driver for Powerflex connectivity will be determined by connection to default array.
 
-### Not Tested But Assumed to Work
+>Note:
 
-* Deployments with the above volume types, provided two pods from the same deployment do not reside on the same node. At the current time anti-affinity rules should be used to guarantee no two pods accessing the same volumes are scheduled to the same node.
-
-### Not Yet Tested or Supported
+The following scenarios are not supported.
 
 * Pods that use persistent volumes from multiple CSI drivers. This _cannot_ be supported because multiple controller-podmons (one for each driver type) would be trying to manage the failover with conflicting actions.
 
-* ReadWriteMany volumes. This may have issues if a node has multiple pods accessing the same volumes. In any case once pod cleanup fences the volumes on a node, they will no longer be available to any pods using those volumes on that node. We will endeavor to support this in the future.
+* Resiliency is not supported for ReadWriteMany volumes in CSM. If a node has multiple pods accessing the same volumes, once a pod cleanup occurs, the volumes will be fenced on that node and will no longer be available to any pods using those volumes
```

Comment:
Change "will be" to "are".  Change "will no longer" to "are no longer".

Reply:
Noted @rsedlock1958 , fixed the same.